apps/system/lockscreen/lockscreen_inputpad_frame.html

```diff
@@ -9,6 +9,8 @@
 
   <div id="lockscreen-panel-passcode" class="lockscreen-panel" data-l10n-id="panel-passcode">
     <p id="lockscreen-passcode-code"><span></span><span></span><span></span><span></span></p>
+    <img src="lockscreen/style/images/lockscreen_unlock.png" id="lockscreen-passcode-lock">
+    <p id="lockscreen-pincode-error-message" hidden="" data-l10n-id="incorrectPinCode">Incorrect PIN Code</p>
```

Comment:
We don't need the input pad for red square. I'd expect you to be deleting this files, not adding new lines to it.